stingray/lightcurve.py

```diff
@@ -264,11 +265,6 @@ def __init__(
                 "Sorry for the inconvenience."
             )
 
-        if err is not None:
-            err = np.asarray(err)
-            if not skip_checks and not np.all(np.isfinite(err)):
-                raise ValueError("There are inf or NaN values in " "your err array")
```

Comment:
Why did this all get removed? We still need to do something with the `err`, right, even if the check for finite values in the GTIs is done elsewhere?

Reply:
Oh, just because this was a duplication. It is already done during the optional checks run if `skip_checks` is `True`